### ISSUE TITLE
Single date select "hint"

### ISSUE THREAD

**Issue by jlebrech** [monterail/angular-date-range-picker#36]
I'd like to put labels that say "please select start date", "please select end date".

I was trying to figure out why clicking one day only didn't change the date.

**Comment by hodak** [comment]
Thanks for opening the issue. We're sorry to say that we've stopped maintaining this project in favor of [angular-mighty-datepicker](https://github.com/monterail/angular-mighty-datepicker). We encourage you to check it out instead.

**Comment by jlebrech** [comment]
do you have examples for picking date ranges with mighty date picker

need something like this:

​It's also making future dates unpickable, but i'd like month in the future
not display.

Thanks.

**Comment by hodak** [comment]
@jlebrech It seems like you tried to attach something but I can't see anything?

**Comment by jlebrech** [comment]
Looks like this:

![screen shot 2015-08-06 at 10 15 12](https://cloud.githubusercontent.com/assets/112134/9109382/22909742-3c2d-11e5-98e0-a808a0c1b166.png)

**Comment by hodak** [comment]
Yes, the last example in `test/index.html` is range picker implemented with mighty picker.

I think you should look at `before / after` attributes which create limits for which days are selectable. Also, `month` option lets you specify number of months used in datepicker.

**Comment by jlebrech** [comment]
thanks I may move to mighty-picker soon
